src/pretix/control/forms/event.py

```diff
@@ -364,7 +366,7 @@ def __init__(self, *args, **kwargs):
             choices=((c.identifier, c.verbose_name) for c in get_all_sales_channels().values()),
             widget=forms.CheckboxSelectMultiple,
         )
-
+    
```

Comment:
Don't leave redundant whitespace. Configure your editor to automatically clean trailing whitespace.